schema.sql

```diff
@@ -10,4 +10,29 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals 
-	ADD species TEXT;
\ No newline at end of file
+	ADD species TEXT;
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name TEXT,
```

Comment:
- [ ] Great job so far. As per project requirement, the data type to use here is a string. `varchar` is most preferable string for storing names. `TEXT` will occupy more space of which it is for storing just name.
**Example**
` name varchar(255),`

Reply:
@t-yanick 

Hey, I want to clarify something regarding the suggested changes. The project requirements specifically ask for the initialization of a column with a string, and that's precisely what I did by selecting TEXT for this purpose. TEXT, along with CHAR() and VARCHAR(), falls under the optional suggestions for initializing a column with a string. Therefore, kindly consider this information and avoid halting the project approval based on changes that were already in line with the requirement section.

Thank you for understanding.